### PR TITLE
Split src/partition

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/repart/meson.build [hunk 1]
```diff
@@ -49,13 +41,13 @@ executables += [
 ]
 
 if conf.get('ENABLE_REPART') == 1
```
Comment (YHNdnzj) [inline src/repart/meson.build:43]: The check can be inverted (check ENABLE_REPART at the beginning of file, if disabled exit early with subdir_done())

Comment (keszybz) [reply src/repart/meson.build:43]: It's just a few lines… I don't see a particular benefit in this case.

Comment (yuwata) [reply src/repart/meson.build:43]: I like the suggestion by @YHNdnzj, but not a big matter here. I think it is OK to merge this as is. @YHNdnzj If you'd like, please create a follow-up PR.

**Review by yuwata — APPROVED** [review]

File: meson.build [hunk 2]
```diff
@@ -2395,12 +2395,12 @@ subdir('src/nss-mymachines')
 subdir('src/nss-resolve')
 subdir('src/nss-systemd')
 subdir('src/oom')
-subdir('src/partition')
 subdir('src/path')
 subdir('src/pcrextend')
 subdir('src/pcrlock')
 subdir('src/portable')
 subdir('src/pstore')
+subdir('src/repart')
```
Comment (yuwata) [inline meson.build]: Not sorted.

**Pull request merged**